diffsync/__init__.py

```diff
@@ -404,13 +404,15 @@ class DiffSync:  # pylint: disable=too-many-public-methods
     # modelname1 = MyModelClass1
     # modelname2 = MyModelClass2
 
-    type: ClassVar[Optional[str]] = None
+    type: Optional[str] = None
```

Comment:
Was this wrong before, or is it a breaking change? I'd thought this *should* be a class variable rather than an instance variable?

Reply:
`self.type` was being set in `__init__`, mypy flagged this as a violation.